locales/en-US/messages.properties

```diff
@@ -220,6 +226,71 @@ momentJSLastEdited=Edited <% timeElapsed %>
+tutorialFeatureTitle=Follow &amp; create embedded tutorials
+tutorialFeatureDesc=Projects can have tutorials that help you (or your students) learn as they make changes to the projects.
+autocompleteFeatureTitle=Get code hints as you type
+autocompleteFeatureDesc=Don't get stuck trying to remember a lot of rules, Thimble will help you along the way for HTML, CSS and even JavaScript files.
+inlineCSSFeatureTitle=Edit CSS right in your HTML pages
+inlineCSSFeatureDesc=Write CSS styles for your elements without switching away from the HTML page that you are editing. Put the text cursor on any element tag and hit <span class="key">Cmd/Ctrl</span> + <span class="key">E</span> to start writing CSS styles.
+publishingFeatureTitle=Publish projects to the web fast
+publishingFeatureDesc=You can publish your projects to the web with only a single click. Later on, when you make changes, you can easily update your project.
+colorpickerFeatureTitle=Pick colors right in the editor
+colorpickerFeatureDesc=Put the text cursor on any color value in a CSS file and press <span class="key">Cmd/Ctrl</span> + <span class="key">E</span> to open the color picker.
+previewModesFeatureTitle=Test your site on a mobile device
+previewModesFeatureDesc=Switching to the mobile preview mode lets you see how your project would look on a mobile phone. You can also turn on a full-screen preview to get the big picture.
+inspectorFeatureTitle=Find HTML elements fast
+inspectorFeatureDesc=Turn on the <strong>DOM Inspector</strong> and hover over different elements in the preview. Thimble will highlight the related code in the HTML editor.
+imageFiltersFeatureTitle=Image Filters
+imageFiltersFeatureDesc=Add some fun to your project by adding filters to your images. Cats never looked so good before!
+takeASelfieFeatureTitle=Take a Selfie
+takeASelfieFeatureDesc=Add your mugshot to any page your're working on using Thimble's selfie tool&mdash;it uses your computer's webcam. Just start typing <span class="key">&lt;img src=</span> tag to your HTML page and you'll see a prompt.
+
+#######################
+## Get Involved page ##
+#######################
+
+getInvolvedHeader=Curious about Open Source? <br/>Help us build &amp; improve Thimble!
+fileAnIssueBtn=File an issue
+goodFirstBugBtn=Find a “Good First Bug” to work on
+whyContributeTitle=Why Contribute?
+contributeReason1=Your work impacts users all over the world
+contributeReason2=Gain valuable experience and mentorship
+contributeReason3=Support Mozilla's mission of an open internet for all
+contributeReason4=It feels good!
+
+getInvolvedDescription1=Thimble is an <a href="https://en.wikipedia.org/wiki/Open-source_software">Open Source</a> project created by the <a href="#">Mozilla Foundation</a> in partnership with <a href="http://cdot.senecacollege.ca/">CDOT at Seneca College</a>. Since the beginning, contributors from all over the world have played an important role in making it a success.
```

Comment:
Ideally, we should use variables for links, and only keep the Wikipedia one hardcoded, since localizers will want to change it to the Wikipedia article in their language if it’s available. Using variables allow us to update links without invalidating existing translations